engine/g5k_engine.py

```diff
@@ -90,10 +91,16 @@ def get_job(self):
 
         self.oarjobs = EX5.get_oargrid_job_oar_jobs(self.gridjob)
 
+        # TODO - Start_date is never used, deadcode ? Ad_rien_ - August 11th 2016
```

Comment:
Yes It seems so, but maybe it's legacy code used by @asimonet that could be used to specify the start_date of a desired reservation